data8-jhub/templates/hub.yaml

```diff
@@ -0,0 +1,170 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+  name: gce-ssd-storage
+provisioner: kubernetes.io/gce-pd
+parameters:
+  type: pd-ssd
```

Comment:
I know that we will probably be deploying to GKE for this semester, but would it be a good idea to template the storage provisioned and parameters as well? So if the deployment is used on a different cloud provider like Azure we can just change the `values.yaml` file instead.